project/src/components/review-list/review-list.tsx

```diff
@@ -14,40 +14,14 @@ export default function ReviewList({reviews}: ReviewListProps): JSX.Element {
         <span className="reviews__amount">{reviews.length}</span>
       </h2>
       <ul className="reviews__list">
-        { reviews.map((review) => (
-          <li className="reviews__item" key={review.id}>
-            <div className="reviews__user user">
-              <div className="reviews__avatar-wrapper user__avatar-wrapper">
-                <img
-                  className="reviews__avatar user__avatar"
-                  src={review.user.avatarUrl}
-                  width="54"
-                  height="54"
-                  alt={review.user.name}
-                />
-              </div>
-              <span className="reviews__user-name">
-                {review.user.name}
-              </span>
-            </div>
-            <div className="reviews__info">
-              <div className="reviews__rating rating">
-                <div className="reviews__stars rating__stars">
-                  <span style={{width: calculateRatingWidth(review.rating)}}></span>
-                  <span className="visually-hidden">Rating</span>
-                </div>
-              </div>
-              <p className="reviews__text">
-                {review.comment}
-              </p>
-              <time className="reviews__time" dateTime={review.date}>
-                {formatDate(review.date)}
-              </time>
-            </div>
-          </li>
+        {reviews && reviews.map((review) => (
```

Comment:
У нас в пропсе заявлено что ревью будут всегда, не нужды обрабатывать undefined, если будет ошибка сервера мы это обработаем 
```suggestion
        {reviews.map((review) => (
```